synapse/lib/stormlib/json.py

```diff
@@ -1,7 +1,76 @@
+import copy
+import json
+import asyncio
+import logging
+
 import synapse.exc as s_exc
+
+import synapse.lib.coro as s_coro
+import synapse.lib.config as s_config
 import synapse.lib.stormtypes as s_stormtypes
 
-import json
+logger = logging.getLogger(__name__)
+
+def runJsSchema(schema, item):
+    # This is a target function for s_coro.forked
+    func = s_config.getJsValidator(schema)
```

Comment:
Do you think at this point getJsSchema / getJsValidator have graduated from config and belong in common?

Reply:
Eh, now looking at the caching bits n/m

Reply:
Possibly? Its not crucial at the moment.